### ISSUE TITLE
LaTeX issue on CentOS

### ISSUE THREAD

**Issue by JamesW38** [part-cw/lambdanative#169]
When I try to compile DemoHelloWorld on CentOS, I experience the following error. What should I do to fix this?

[maxnchz@deimos lambdanative]$ ./configure DemoHelloWorld android debug verbose
==> configured to build DemoHelloWorld for android in normal mode
 == using source in /home/maxnchz/lambdanative/apps/DemoHelloWorld

[maxnchz@deimos lambdanative]$ make
==> creating artwork needed for DemoHelloWorld..
==> creating textures needed for DemoHelloWorld..
==> creating fonts needed for DemoHelloWorld..
==> creating strings needed for DemoHelloWorld..
 => title..
xelatex tmp.tex
This is XeTeXk, Version 3.141592-2.2-0.996 (Web2C 7.5.6)
 %&-line parsing enabled.
entering extended mode
(./tmp.tex
LaTeX2e <2005/12/01>
Babel <v3.8h> and hyphenation patterns for english, usenglishmax, dumylang, noh
yphenation, arabic, basque, bulgarian, coptic, welsh, czech, slovak, german, ng
erman, danish, esperanto, spanish, catalan, galician, estonian, farsi, finnish,
 french, greek, monogreek, ancientgreek, croatian, hungarian, interlingua, ibyc
us, indonesian, icelandic, italian, latin, mongolian, dutch, norsk, polish, por
tuguese, pinyin, romanian, russian, slovenian, uppersorbian, serbian, swedish,
turkish, ukenglish, ukrainian, loaded.

Output file removed.
ERROR: failed on file tmp.pdf
BUILD FAILED
make: *** [all] Error 1

Thanks!

**Comment by mgorges** [comment]
Am I right to assume that you only pasted the relevant pieces of the output? In that case can we have the full output as a [Gist](https://gist.github.com), please? - Your problem is interesting as by the time you get to this point, where the string files are rendered, the `xelatex` check should already have run successfully, so I would be interested to see the results of that piece as well.

**Comment by JamesW38** [comment]
You are correct. Thanks for the swift response. Here is a Gist containing the complete output: https://gist.github.com/maxnchz/d1022316092746da3ff90822e411dd7f

**Comment by mgorges** [comment]
This piece is puzzling, maybe it is a font issue? - But the cmss.ttf, which is used to render the title strings is included so this might be unrelated.
> ** NOTICE: This document contains a `Preview & Print only' licensed font **
--> Will have to think about this a bit more tomorrow.

**Comment by mgorges** [comment]
Out of curiosity, will DemoHelloWorld build for your native platform, Linux?

**Comment by JamesW38** [comment]
I get the exact same result whether I build for Linux or Android. I also think it is related to font's, ie DemoHelloWorld's use of glgui-button-string calling ascii_18.fnt. That would explain why I can build the Calculator example app just fine.

**Comment by mgorges** [comment]
That last piece is helpful information - will look into it more tomorrow. Can you try using the calculator fonts in DemoHelloWorld?

**Comment by JamesW38** [comment]
Perhaps I am misunderstanding, but it doesn't look like the Calculator app makes explicit use of any fonts.

**Comment by ddunsmuir** [comment]
The Calculator app does use the font DejaVuSans.ttf. See the FONTS file which defines the three different font sizes with it and calls them calc. Thus, when it refers to "calc_32.fnt" for example, it is using the DejaVuSans.ttf font, size 32. James, can you try using DejaVuSans font in DemoHelloWorld?

**Comment by peterlew** [comment]
@JamesW38 are you still interested in resolving this issue? I'd like to help, if possible.
